Wayfindr Demo/Classes/Controller/Maintainer/BatteryLevelsTableViewController.swift

```diff
@@ -26,26 +26,39 @@
 import UIKit
 
 import SVProgressHUD
+// FIXME: comparison operators with optionals were removed from the Swift Standard Libary.
```

Comment:
You should create a GitHub issue for this and include a link in your FIXME comment to track and make sure it does get fixed.